ui/src/pages/Lore/Lore.tsx

```diff
@@ -31,32 +32,41 @@ export function Lore() {
     };
     fetchData();
   }, []);
-  console.log(lore[currentLoreSelection])
+
+  const isPlainText = () => {
+    setPlainText(!plainText);
+  }
+
+  let text = (depth: string) => plainText ? 'lore__no_effect' : 'lore__container' + [depth];
+
   return (
     <div className="lore">
       {loading ? (
         <> loading </>
       ) : (
-      <div className="lore__container">
-      <div id="fade"></div>
-        <div className="lore__container__effect">
-          <div className="lore__container__effect__text">
-            <h3 className="header3">LOCATION:</h3>
-            <p className="paragraph">{lore[currentLoreSelection].location}</p>
-            <h3 className="header3">Character:</h3>
-            <p className="paragraph">{lore[currentLoreSelection].character}</p>
-            <h3 className="header3">Background:</h3>
-            {sectionedBackground.map((section) => (
-              <div key={section}>
-                <p className="paragraph">
-                  {section}
-                </p>
-                <br />
+        <>
+          <button type="button" onClick={isPlainText}>PRESS TO CHANGE</button>
+          <div className={text("")}>
+            {plainText ? null : <div id="fade"/>}
+            <div className={text("__effect")}>
+              <div className={text("__effect__text")}>
```

Comment:
I dont like this way of doing this. Instead of `__effect__text`. Just put the beginning `__` inside of the turnary
`let text = (depth: string) => plainText ? 'lore__no_effect' : 'lore__container__' + [depth];`

Reply:
No because that messes with the first level `lore__container`. If I add the underscores into the function then `lore__container__` will become the problem.

Reply:
Heres what we can do, after messing around with the code a little bit, looks like we can just straight up get rid of the function call inside of the classname and use 
``` <div className="lore__container">```
for the top most level and it would work as normal. Then we can proceed to do what with i said and do `lore__no_effect' : 'lore__container__'` and have it be `<div className={text("effect")}>`. Just tested it on my machine, looks good to me